app/stores/country-list.js

```diff
@@ -0,0 +1,2 @@
+import ListStore from 'focus-core/store/list';
+export default new ListStore({identifier: 'countryList'});
```

Comment:
Property is named différently than its value... That not helps understanding or debbuging. No ?